gnovm/stdlibs/std/banker.gno

```diff
@@ -126,12 +131,26 @@ func (b banker) IssueCoin(addr Address, denom string, amount int64) {
 	if b.bt != BankerTypeRealmIssue {
 		panic(b.bt.String() + " cannot issue coins")
 	}
+	assertRealmDenom(denom)
 	bankerIssueCoin(uint8(b.bt), string(addr), denom, amount)
 }
 
 func (b banker) RemoveCoin(addr Address, denom string, amount int64) {
 	if b.bt != BankerTypeRealmIssue {
 		panic(b.bt.String() + " cannot remove coins")
 	}
+	assertRealmDenom(denom)
 	bankerRemoveCoin(uint8(b.bt), string(addr), denom, amount)
 }
+
+func assertRealmDenom(denom string) {
+	prefix := "/" + CurrentRealm().PkgPath() + ":"
+	if !strings.HasPrefix(denom, prefix) {
+		panic("cannot issue coins with denom not prefixed by: /realmPath:")
+	}
+
+	baseDenom := strings.TrimPrefix(denom, prefix)
```

Comment:
since we use HasPrefix before, we can just use `denom[len(prefix):]`  here

Reply:
[903e06e](https://github.com/gnolang/gno/pull/3239/commits/903e06efb14e905d2620eaaace69f4f8249654da)